packages/libs/eda/src/lib/core/components/visualizations/implementations/VolcanoPlotVisualization.tsx

```diff
@@ -152,6 +160,130 @@ function VolcanoPlotViz(props: VisualizationProps<Options>) {
     ])
   );
 
+  /**
+   * Find mins and maxes of the data and for the plot.
+   * The standard x axis is the log2 fold change. The standard
+   * y axis is -log10 raw p value.
+   */
+
+  // Find maxes and mins of the data itself
+  const rawDataMinMaxValues: RawDataMinMaxValues = useMemo(() => {
+    if (!data.value)
+      return {
+        x: { min: 0, max: 0 },
+        y: { min: 1, max: 1 },
```

Comment:
NOTE: I changed the fallback y-values to be 1 so that the empty plot doesn't get truncated. Previously, the values were 0 and `Math.log10(0)` returns `-Infinity`, resulting in the yMax truncation bar rendering on the empty graph placeholder.